store/modules/actions/authCore.js

```diff
@@ -126,15 +127,41 @@ export async function fetchAuthCoreCosmosWallet({ state }) {
   return cosmosAddress;
 }
 
-export async function prepareAuthCoreCosmosTxSigner({ state }) {
-  if (!state.cosmosProvider) throw new Error('COSMOS_WALLET_NOT_INITED');
-  return async function signer(signMessage) {
-    const data = JSON.parse(signMessage);
-    const dataWithSign = await state.cosmosProvider.sign(data);
-    const signObject = dataWithSign.signatures[dataWithSign.signatures.length - 1];
```

Comment:
I think only  the part `return async function signer(signMessage) {` needs to be change, should not add `setupCosmosProvider` flow as it is already implement above as `initAuthCoreWalletService`, and the old flow does not handle reauth